### PR TITLE
Custom project structure

### LINKED ISSUE #9: Document custom structure (#565)
Document https://github.com/versionpress/versionpress/issues/565. There are only some supported scenarios of customising project structure. 
- [ ] Document supported scenarios
- [ ] Document public "VP_" constants, e.g., `VP_PROJECT_ROOT`, `VP_VPDB_DIR`.

### PR DESCRIPTION
**Pull request by JanVoracek**
Resolves #9.

Reviewers:
- [x] @borekb

### CODE REVIEW

File: content/en/01-getting-started/05-configuration.md [hunk 1]
```diff
@@ -55,9 +55,14 @@ By default, VersionPress calls just `git` which leaves the path resolution up to
 
 ### VP_PROJECT_ROOT
 
-[TODO]
+*Default: `ABSPATH`*  
+*Configuration file: `wp-config.common.php`*
+
+By default, VersionPress creates the repository (and the `.git` directory) in the `ABSPATH` directory. If you [move WordPress into its own directory](../feature-focus/custom-project-structure#giving-wordpress-its-own-directory), you have to define this constant to reflect the original `ABSPATH`.
```
Comment (borekb) [inline content/en/01-getting-started/05-configuration.md]: > you have to define this constant to reflect the original `ABSPATH`.

This is not entirely clear to me, perhaps an example would help here.

File: content/en/02-feature-focus/08-custom-project-structure.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+since: 3.0
+---
+
+# Custom project structure
+
+Some advanced users like having WordPress in its own directory or move plugins, themes or uploads in another directory. VersionPress supports some scenarios.
```
Comment (borekb) [inline content/en/02-feature-focus/08-custom-project-structure.md]: We should stress that the directories must still be under the "project root" (where the Git repo is maintained).

**Comment by JanVoracek** [comment]
@borekb: I made some modifications. Can you review this again?

**Comment by borekb** [comment]
Looks good.

**Pull request merged**